conans/client/generators/virtualenv_python.py

```diff
@@ -7,6 +8,7 @@ class VirtualEnvPythonGenerator(VirtualRunEnvGenerator):
     venv_name = "conanenvpython"
 
     def __init__(self, conanfile):
+        conan_v2_behavior("'virtualenv_python' generator is deprecated")
```

Comment:
This probably needs to stay, until we fully have a replacement for Conan 2.0.

It was added recently in https://github.com/conan-io/conan/pull/5511

Seems that trying to centralize the virtualenv generators would be a possible replacement.